modules/01-mods.js

```diff
@@ -280,13 +280,20 @@ async function slashModSlowmode(interaction) {
 
   if (duration <= 0) {
     ch.edit({ rateLimitPerUser: 0 }).catch(e => u.errorHandler(e, interaction));
-
+    const old = molasses.get(ch.id);
     if (molasses.has(ch.id)) {
       clearTimeout(molasses.get(ch.id));
       molasses.delete(ch.id);
```

Comment:
May as well use the `old` variable you created in these lines, to DRY things up

Reply:
Fixed in latest commit